config.yaml

```
@@ -41,13 +41,66 @@ imaging:
 languages:
   en:
     title: etcd
-    languageName: English
+    languageName: 🇬🇧 English
```

I suggest using the ISO code of the language instead of the flag
```suggestion
    languageName: English (en)
```